fn_guardium_insights_integration/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
see https://github.ibm.com/Resilient/resilient-community-apps/blob/master/fn_outbound_email/entrypoint.sh for updates.

Reply:
Done